### PR TITLE
Smarthub replace placement

### PR DESCRIPTION
**Pull request by SmartHubSolutions**
## Type of change

- [ ] Bugfix
- [ ] Feature
- [ ] New bidder adapter  
- [x] Updated bidder adapter   
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes

- [ ] Does this change affect user-facing APIs or examples documented on http://prebid.org?
- [ ] Other

### CODE REVIEW

**Review by jsnellbaker — CHANGES_REQUESTED** [review]
@SmartHubSolutions 
While these changes look fine, are there any unit tests that should be updated to reflect these changes?

File: modules/smarthubBidAdapter.js [hunk 1]
```diff
@@ -61,31 +61,31 @@ function getPlacementReqData(bid) {
...
-    placement.api = mediaTypes[VIDEO].api;
-    placement.linearity = mediaTypes[VIDEO].linearity;
+    plcmt.adFormat = VIDEO;
+    plcmt.playerSize = mediaTypes[VIDEO].playerSize;
+    plcmt.minduration = mediaTypes[VIDEO].minduration;
+    plcmt.maxduration = mediaTypes[VIDEO].maxduration;
+    plcmt.mimes = mediaTypes[VIDEO].mimes;
+    plcmt.protocols = mediaTypes[VIDEO].protocols;
+    plcmt.startdelay = mediaTypes[VIDEO].startdelay;
+    plcmt.placement = mediaTypes[VIDEO].placement;
```
Comment (patmmccann) [inline modules/smarthubBidAdapter.js:74]: This is the only line that matters here, you must also add placement or plcmt.plcmt

**Review by patmmccann — CHANGES_REQUESTED** [review]

File: modules/smarthubBidAdapter.js (hunk 1 above)
Comment (SmartHubSolutions) [reply modules/smarthubBidAdapter.js]: resolved

**Comment by SmartHubSolutions** [comment]
> @SmartHubSolutions While these changes look fine, are there any unit tests that should be updated to reflect these changes?

test are updated

**Review by jsnellbaker — APPROVED** [review]

**Review by patmmccann — APPROVED** [review]

**Pull request merged**